### ISSUE TITLE
`hide` derive doesn't hide commands

### ISSUE THREAD

**Issue by ematipico** [pacak/bpaf#214]
I have a niche case where I have some commands that we deem "internal", so they need to be parsed by `bpaf` but we don't want to expose them to the user.

I have some code like this:

```rs
#[derive(Bpaf)]
enum Commands {
	#[bpaf(command)]
	Foo,
	#[bpaf(command, hide)]
	Private
}
```

Although, when printing the `--help` command, the command appears. Is it a bug, or is it intentional? Or a missing feature?

**Comment by pacak** [comment]
Looks like a bug to me. Let me take a closer look.

**Comment by pacak** [comment]
Yes, it's a bug in `bpaf_derive`, `hide` is ignored. Let me fix that real quick....

**Comment by ematipico** [comment]
Thank you for the prompt reply! :)

**Comment by pacak** [comment]
> Thank you for the prompt reply! :)

Fixed and pushed to `hidden_command` branch. Can you check if it deals with your problem?

**Comment by ematipico** [comment]
> > Thank you for the prompt reply! :)
> 
> Fixed and pushed to `hidden_command` branch. Can you check if it deals with your problem?

Yes, it works!

**Comment by pacak** [comment]
Released 0.8.1 in #215
